src/index.js

```diff
@@ -130,6 +130,10 @@ class LumigoPlugin {
 		const functions = service
 			.getAllFunctions()
 			.filter(localName => functionNames.includes(localName))
+			.filter(localName => {
+				const { lumigo = {} } = this.serverless.service.getFunction(localName);
+				return typeof lumigo.enabled === "undefined" || lumigo.enabled === true;
```

Comment:
Why `typeof`?

```suggestion
				return lumigo.enabled === "undefined" || lumigo.enabled === true;
```

Reply:
The latter will return true for:

```yml
lumigo:
  enabled: undefined
```

Reply:
@roni-frantchi I see, then why not using ?
```suggestion
				return lumigo.enabled === undefined || lumigo.enabled === true;
```

Reply:
@efimk-lu see https://github.com/lumigo-io/serverless-lumigo-plugin/pull/36/commits/3f15e0a63fd58ed13bb2aede6c722deb9f0d2da0
Note I used `==` and not `===` because `undefined == null && undefined !== null` and I'd rather capture those too.

Reply:
I prefer explicit
`lumigo.enabled === undefined || lumigo.enabled === null || ...`

The fact that you needed to explain to me shows that something is smelly here

Reply:
@efimk-lu 
Coming from other languages, one might argue that on
‘’’javascript
const foo = true;
...
if(foo) // why not foo === true?
‘’’
it’s **very** common practice in JavaScript.